db/schema.rb

```diff
@@ -37,6 +37,16 @@
   add_index "districts", ["county_id"], name: "index_districts_on_county_id", using: :btree
   add_index "districts", ["educational_service_district_id"], name: "index_districts_on_educational_service_district_id", using: :btree
 
+  create_table "dropouts", force: :cascade do |t|
+    t.integer  "year_1"
+    t.integer  "year_2"
+    t.integer  "year_3"
+    t.integer  "year_4"
+    t.integer  "year_5"
```

Comment:
So I was wondering if there was a better way to go about this. For example I was thinking it would be neat to have one column that was a hash and could store all the years together. Turns out there is and it's called store! Though it would sacrifice being able to do query calls on it. Oh well still neat to know. 
http://api.rubyonrails.org/classes/ActiveRecord/Store.html